webapp/components/interop-dashboard.js

```diff
@@ -727,51 +805,77 @@ class InteropDashboard extends PolymerElement {
     for (let i = 0; i < rowNames.length; i++) {
       featureOrder[i] = rowNames[i][0];
     }
-  }
+  };
 
-  numericalSort(rows, featureOrder, sortColumn) {
+  numericalSort = (rows, featureOrder, sortColumn) => {
+    const browserIndex = sortColumn - 1;
     const individualScores = [];
     for (let i = 0; i < rows.length; i++) {
       const feature = rows[i];
-      individualScores[i] = [feature, this.getNumericalBrowserScoreByFeature(sortColumn, feature)];
+      individualScores[i] = [feature, this.getNumericalBrowserScoreByFeature(browserIndex, feature)];
     }
     individualScores.sort((a, b) => a[1] - b[1]);
     for (let i = 0; i < individualScores.length; i++) {
       featureOrder[i] = individualScores[i][0];
     }
-  }
+  };
 
-  sortRows(rows, index, sortColumn, isSortedAsc) {
+  sortRows = (rows, index, sortColumn, isSortedAsc) => {
     if(index !== 0) {
       return rows;
     }
     const sortedFeatureOrder = [];
     // For the first column, sort alphabetically by name
-    if(sortColumn === -1) {
+    if(sortColumn === 0) {
       this.alphabeticalSort(rows, sortedFeatureOrder);
+    } else {
       // For the other columns, sort numerically by score
-    } else if (sortColumn >= 0) {
       this.numericalSort(rows, sortedFeatureOrder, sortColumn);
     }
+
     // Reverse current sort order
     if (!isSortedAsc) {
       sortedFeatureOrder.reverse();
     }
     return sortedFeatureOrder;
-  }
-
-  handleSortClick(e) {
-    const i = parseInt(e.target.id.split('-')[1]) - 1;
-
-    if (this.sortColumn !== i) {
-      this.sortColumn = i;
-      this.isSortedAsc = true;
-    } else if (this.sortColumn === i && this.isSortedAsc) {
+  };
+
+  /**
+   * Column sort handlers.
```

Comment:
Optional: You can rewrite the HTML with dom-repeat and pass the index as i

Reply:
I thought this as well, and it was my first approach to the implementation, but could not figure out how to do this in the `on-click` while passing the index, and I think we haven't passed arguments into an `on-click` function anywhere else in the app. But I do assume that there is a better way of doing this.